React-native-camera package fails to build

Steps to reproduce:

* Download `react-native-camera'
* Init project
* Run project using `yarn android`

FAILURE: Build failed with an exception.
* What went wrong:
Failed to capture snapshot of input files for task ':react-native-camera:compileReleaseAidl' property 'importDirs' during up-to-date check.
> The library com.google.android.gms:play-services-basement is being requested by various other libraries at [[15.0.1,15.0.1]], but resolves to 12.0.1. Disable the plugin and check your dependencies tree using ./gradlew :app:dependencies.

--- a/packages/flagship/src/lib/modules/react-native-camera.ts
+++ b/packages/flagship/src/lib/modules/react-native-camera.ts
@@ -1,0 +1,29 @@
+import * as path from '../path';
+import * as fs from '../fs';
+import { Config } from '../../types';
+import {
+  logInfo
+} from '../../helpers';
+
+export function android(configuration: Config): void {
+  // Add dependencies to /android/app/build.gradle and replace compile command with implementation
+  // command due to Gradle 3 changes
+  let gradleAppBuild = fs.readFileSync(path.android.gradlePath(), { encoding: 'utf8' });
+
+  gradleAppBuild = gradleAppBuild.replace(/compile /g, 'implementation ');
+  gradleAppBuild = gradleAppBuild.replace(/compile\(/g, 'implementation(');
+  fs.writeFileSync(path.android.gradlePath(), gradleAppBuild);
+  logInfo('updated ./android/app/build.gradle');
+
+  // Add dependencies to /android/build.gradle and update gradle version to 3.1.3
+  let gradleBuild = fs.readFileSync(path.resolve('android', 'build.gradle'), { encoding: 'utf8' });
+  gradleBuild = gradleBuild.replace(
+    /(com\.android\.tools\.build:gradle:)[\d\.]+/,
+    '$13.1.3'
+  );
+
+  fs.writeFileSync(path.resolve('android', 'build.gradle'), gradleBuild);
+  logInfo('updated ./android/build.gradle');
+
+  logInfo('finished updating Android for react-native-camera');
+}

